fixtures/public/react-loader.js

```diff
@@ -7,8 +7,8 @@
  * (Loads React 15.4.1)
  */
 
-var REACT_PATH = '../../build/react.js';
-var DOM_PATH = '../../build/react-dom.js';
+var REACT_PATH = 'react.js';
+var DOM_PATH = 'react-dom.js';
```

Comment:
We need to figure out the best way to get a copy of the local `react` and `react-dom` builds.  The relative path was not working out of the box, so I ended up just copying the files from `build` manually. Maybe we should make that part of the build process, or do it before the `start` script for this app?

If you're testing locally, make sure to copy those files into here for now!